workspaces/lightspeed/plugins/lightspeed/src/components/LightSpeedChat.tsx

```diff
@@ -169,29 +174,45 @@ export const LightspeedChat = ({
     })();
   }, [createConversation, setConversationId, setMessages]);
 
+  const openDeleteModal = (conversation_id: string) => {
+    document.dispatchEvent(
```

Comment:
Add a TODO comment to remove this in the future, as it is only a temporary workaround.